commit 198bd4d69dddb3fe78fdc251983bb49502b3b35e
Author: Jason Cooper <jason@lakedaemon.net>
Date:   Tue Sep 14 09:45:42 2010 -0400

    staging: brcm80211: fix "ERROR: need consistent spacing around '*'"
    
    This patch fixes the real errors.  The rest are caused by typedefs
    which will be fixed in a later patch.
    
    Signed-off-by: Jason Cooper <jason@lakedaemon.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/d11.h
+++ b/drivers/staging/brcm80211/include/d11.h
@@ -1201,8 +1201,8 @@ BWL_PRE_PACKED_STRUCT struct tx_status {
 #define M_20IN40_IQ			(0x380 * 2)
 
 /* SHM locations where ucode stores the current power index */
-#define M_CURR_IDX1		(0x384 *2)
-#define M_CURR_IDX2		(0x387 *2)
+#define M_CURR_IDX1		(0x384 * 2)
+#define M_CURR_IDX2		(0x387 * 2)
 
 #define M_BSCALE_ANT0	(0x5e * 2)
 #define M_BSCALE_ANT1	(0x5f * 2)
--- a/drivers/staging/brcm80211/util/bcmsrom.c
+++ b/drivers/staging/brcm80211/util/bcmsrom.c
@@ -503,8 +503,8 @@ BCMATTACHFN(srom_parsecis) (osl_t *osh,
 					    && !(ETHER_ISMULTI(&cis[i + 2]))) {
 						ASSERT(cis[i + 1] ==
 						       ETHER_ADDR_LEN);
-						bcm_ether_ntoa((struct
-								ether_addr *)
+						bcm_ether_ntoa(
+							(struct ether_addr *)
 							       &cis[i + 2],
 							       eabuf);
 
@@ -975,8 +975,8 @@ BCMATTACHFN(srom_parsecis) (osl_t *osh,
 				case HNBU_MACADDR:
 					if (!(ETHER_ISNULLADDR(&cis[i + 1])) &&
 					    !(ETHER_ISMULTI(&cis[i + 1]))) {
-						bcm_ether_ntoa((struct
-								ether_addr *)
+						bcm_ether_ntoa(
+							(struct ether_addr *)
 							       &cis[i + 1],
 							       eabuf);
 
